src/test/java/com/wikia/webdriver/testcases/seotests/HtmlTitleTests.java

```diff
@@ -0,0 +1,206 @@
+            false
+        },
+        {
+            "sktest123",
+            "Style-5H2?vaction=edit",
+            "Style-5H2 - Sktest123 Wiki - Wikia",
+            false
+        },
+        {
+            "sktest123",
+            "Style-5H2?action=history",
+            "Revision history of \"Style-5H2\" - Sktest123 Wiki - Wikia",
+            false
+        },
+        {
+            "sktest123",
+            "Special:Version",
+            "Version - Sktest123 Wiki - Wikia",
+            false
+        },
+        {
+            "sktest123",
+            "Special:Videos",
+            "Videos on this wiki - Sktest123 Wiki - Wikia",
+            false
+        },
+        {
+            "sktest123",
+            "Special:NewFiles",
+            "New photos on this wiki - Sktest123 Wiki - Wikia",
+            false
+        },
+        {
+            "sktest123",
+            "Blog:Recent_posts",
+            "Blog:Recent posts - Sktest123 Wiki - Wikia",
+            false
+        },
+        {
+            "sktest123",
+            "User_blog:Sktest/test_blog_1",
+            "User blog:Sktest/test blog 1 - Sktest123 Wiki - Wikia",
+            false
+        },
+        {
+            "sktest123",
+            "Special:Chat",
+            "Sktest123 Wiki: Welcome to the Sktest123 Wiki chat - Wikia",
+            true
+        },
+        {
+            "sktest123",
+            "Special:Forum",
+            "Forum - Sktest123 Wiki - Wikia",
+            false
+        },
+        {
+            "sktest123",
+            "Board:General_Discussion",
+            "General Discussion board - Sktest123 Wiki - Wikia",
+            false
+        },
+        {
+            "sktest123",
+            "Thread:2610",
+            "Test post - Sktest123 Wiki - Wikia",
+            false
+        },
+        {
+            "sktest123",
+            "Category:Premium_Videos",
+            "Premium Videos - Sktest123 Wiki - Wikia",
+            false
+        },
+        {
+            "sktest123",
+            "Message_Wall:Sktest",
+            "Message Wall:Sktest - Sktest123 Wiki - Wikia",
+            false
+        },
+        {
+            "sktest123",
+            "Thread:2160",
+            "Welcome to Sktest123 Wiki! - Sktest123 Wiki - Wikia",
+            false
+        },
+        {
+            "sktest123",
+            "Special:Maps",
+            "Maps - Wikia",
+            false
+        },
+        {
+            "sktest123",
+            "Template:Welcome",
+            "Template:Welcome - Sktest123 Wiki - Wikia",
+            false
+        },
+        {
+            "sktest123",
+            "MediaWiki:Common.css",
+            "MediaWiki:Common.css - Sktest123 Wiki - Wikia",
+            false
+        },
+        {
+            "sktest123",
+            "MediaWiki:Edit",
+            "MediaWiki:Edit - Sktest123 Wiki - Wikia",
+            false
+        },
+        // custom title
+        {
+            "es.pokemon",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "es.pokemon" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)